soc/arm/nxp_imx/rt6xx/soc.c

```diff
@@ -24,12 +24,15 @@
 #include <fsl_clock.h>
 #include <fsl_common.h>
 #include <fsl_device_registers.h>
+#include "flash_clock_setup.h"
```

Comment:
Would this need to be wrapped inside
`#ifdef CONFIG_CODE_FLEXSPI`

Reply:
Sure, I can do that.